lib/stream/helper/leos3.js

```diff
@@ -55,7 +56,7 @@ module.exports = function(ls, queue, configure, opts, onFlush) {
 
 	function newStream() {
 		var timestamp = moment();
-		let postfix = (pad + (++fileCount)).slice(padLength);
+		let postfix = (pad + (++fileCount)).slice(padLength) + "-" + uuid.v4();
```

Comment:
Add a uuid so that the file name doesn't collide with other processes running concurrently